spec/requests/odata/root_spec.rb

```diff
@@ -20,14 +20,17 @@
     include_context "odata with basic forms"
 
     it "renders as expected" do
-      names = ["Responses: #{form.name}", "Responses: #{form_with_no_responses.name}"]
-      urls = %W[Responses-#{form.id} Responses-#{form_with_no_responses.id}]
+      forms = [form, form_with_no_responses, paused_form]
+      values = forms.map do |form|
+        {
+          name: "Responses: #{form.name}",
+          kind: "EntitySet",
+          url: "Responses-#{form.id}"
+        }
+      end
```

Comment:
Is this stuff supposed to be in this commit?

Reply:
This is just a small refactor after I added a third form `paused_form` to the array here and I wanted to DRY up the logic